docs/introduction.rst

```diff
@@ -65,23 +67,32 @@ in which many of the nodes and individuals have been removed.
 Who and what is in the tree sequence?
 *************************************
 
-OK, who and what exactly *is* left in the tree sequence?
+OK, who and what exactly *is* left in the tree sequence after simplification?
```

Comment:
It migth be confusing to say "after simplification" here: the reader might not know that "simplification" refers to a specific process (the previous paragraph is understandable without thinking of it as a technical term).

Reply:
Maybe "who and what exactly *is* left in the tree sequence after the unnecessary information has been removed?" (or, "has been simplified away"?)

Reply:
Good point. On that note, when you mention simplification later on, in `tutorial.rst`, it's as if you have never mentioned it before. I wonder if, in the tutorial, you should say something like "explicit simplification, in a similar way that the tree sequence is simplified on output, as described in the intro"?

Reply:
Good idea, this should be clarified. In the tutorial I'm talking about running-simplify-in-python, not the simplification that happens in SLiM as it goes along. The reason it's like this is because simplification-by-SLiM is something that I think people can safely ignore: it's how things work under the hood, but I find that if we start talking about simplification a lot then people think *they* have to do it, but things are less error-prone if they don't do it. But clarification would be good.